gnoland/website/views/faucet.html

```diff
@@ -14,14 +14,19 @@
 
 This is the Gno.land (test) faucet.
 
+<script src="https://www.google.com/recaptcha/api.js" async defer></script>
+
+<!-- <form method="POST" action="http://localhost:5050/"> -->
 <form method="POST" action="http://gno.land:5050/">
   <label for="toaddr">GNO.land address (e.g. g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5):</label><br>
   <input type="text" id="toaddr" name="toaddr"/><br>
```

Comment:
perhaps adding required attribute would be better

<input type="text" id="toaddr" name="toaddr" required>